monai/transforms/lazy/functional.py

```diff
@@ -11,51 +11,90 @@
 
```

Comment:
Do we really need this "lazy" layer folder?
In this case, we will also have some "instant" version `functional.py`?

Thanks.

Reply:
maybe we'll need this when we go for a lazy compose implementation... I agree we should review this folder structure before we release the lazy resampling feature.